src/content/peerconnection/audio/index.html

```diff
@@ -45,8 +45,9 @@
       </div>
     </div>
 
-    <div id="buttons">
-      <select id="codec">
+    <div id="params">
+       Codec:
```

Comment:
These labels should be in label elements. (See [constraints example](http://webrtc.github.io/samples/src/content/peerconnection/constraints/).)